doc/modules/feature_selection.rst

```diff
@@ -131,6 +131,26 @@ number of features.
       elimination example with automatic tuning of the number of features
       selected with cross-validation.
 
+.. _select_from_model:
+
+Feature selection using SelectFromModel
```

Comment:
I think it might be more useful to explicitly mention the name of the model in the title.